ios/RNTPTDocumentView.m

```diff
@@ -3018,6 +3018,90 @@ - (void)setPropertiesForAnnotation:(NSString *)annotationId pageNumber:(NSIntege
     }
 }
 
+#pragma mark - Get Annotation(s)
+
+- (NSDictionary *)getAnnotationAtPoint:(NSInteger)x y:(NSInteger)y distanceThreshold:(double)distanceThreshold minimumLineWeight:(double)minimumLineWeight
+{
+    PTPDFViewCtrl *pdfViewCtrl = self.currentDocumentViewController.pdfViewCtrl;
+    PTPDFDoc *pdfDoc = self.currentDocumentViewController.document;
+    
+    __block NSDictionary *annotation;
+    if (pdfViewCtrl && pdfDoc) {
+        NSError *error;
+        
+        [pdfViewCtrl DocLockReadWithBlock:^(PTPDFDoc * _Nullable doc) {
+            PTAnnot *annot = [pdfViewCtrl GetAnnotationAt:(int)x y:(int)y distanceThreshold:distanceThreshold minimumLineWeight:minimumLineWeight];
+            
+            if (annot && [annot IsValid]) {
+                annotation = [self getAnnotationData:annot pageNumber:[[annot GetPage] GetIndex] pdfViewCtrl:pdfViewCtrl];
```

Comment:
`[annot GetPage]` is not reliable and should not be used. Instead, you should use https://www.pdftron.com/api/ios/Classes/PTPDFViewCtrl.html#/c:objc(cs)PTPDFViewCtrl(im)GetPageNumberFromScreenPt:y: to get the page number. (and use https://www.pdftron.com/api/ios/Classes/PTPDFDoc.html#/c:objc(cs)PTPDFDoc(im)GetPage: if you ever need the page object.)